test/queries/getUsersByInterest.test.js

```diff
@@ -0,0 +1,19 @@
+const test = require('tape');
+const dbBuild = require('../../server/database/config/dbbuild');
+const { users: { getUsersByInterest } } = require('../../server/database/queries');
+
+test('Testing for the users by interest query', (t) => {
+  const expected = [{
+    interest_id: 1, user_id: 1, username: 'amoodaa', avatar_path: null, bio: null, native_lang_id: 1, learning_lang_id: 6,
```

Comment:
make your code here more readable :eyes: 
```js
 const expected = [{
    interest_id: 1,
    user_id: 1,
    username: 'amoodaa',
    avatar_path: null,
    bio: null,
    native_lang_id: 1,
    learning_lang_id: 6,
  }, {
    interest_id: 1,
    user_id: 2,
    username: 'fadi',
    avatar_path: null,
    bio: null,
    native_lang_id: 1,
    learning_lang_id: 5,
  }];

```
just suggestion to be your code cleaner and more readable